agent/acs/handler/attach_eni_handler.go

```diff
@@ -118,14 +129,36 @@ func (attachENIHandler *attachENIHandler) handleSingleMessage(message *ecsacs.At
 // addENIAttachmentToState adds the eni info to the state
 func (handler *attachENIHandler) addENIAttachmentToState(message *ecsacs.AttachTaskNetworkInterfacesMessage) {
 	attachmentArn := aws.StringValue(message.ElasticNetworkInterfaces[0].AttachmentArn)
+	eniMac := aws.StringValue(message.ElasticNetworkInterfaces[0].MacAddress)
+	// Stop tracking the eni attachment after timeout
+	eniAckTimeout := time.Duration(aws.Int64Value(message.WaitTimeoutMs)) * time.Millisecond
+	_time := ttime.DefaultTime{}
+	ackTimeoutTimer := _time.AfterFunc(eniAckTimeout, handler.ackTimeoutHandler(eniMac))
```

Comment:
I don't think we need to use `ttime.DefaultTime{}` here (I'd prefer to not use it anywhere). The `time.Timer` struct should satisfy the ttime.Timer interface IIUC.